python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -166,6 +166,90 @@ def test_library_management_path_prefix(moto_s3_uri_incl_bucket, boto_client):
         _lib = ac["pytest_test_lib"]
 
 
+def test_read_meta_batch_with_tombstones(arctic_library):
```

Comment:
I can't see any tests for how this behaves when there is _no_ metadata for a particular symbol?

Reply:
Should consider how the existing `batch_read` behaves for non-existent versions and make sure we are consistent with it.

Reply:
done the test